ros2_batch_job/__main__.py

```diff
@@ -356,6 +370,32 @@ def build_and_test(args, job):
     # return 0 if ret_test == 0 and ret_testr == 0 else 1
     return 0
 
+'''
+Adds sanitizer specific mixin as a build arg
+'''
```

Comment:
The docblock of a function needs to be in the function block and should use triple double quotes.

Same below.

Reply:
Removed all sanitizer specific logic from code